src/o365/outlook/commands/report/outlook-report-getmailboxusagestorage.ts

```diff
@@ -0,0 +1,38 @@
+import commands from '../../commands';
+import PeriodBasedReport from '../../../base/PeriodBasedReport';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+class OutlookGetMailboxUsageStroageCommand extends PeriodBasedReport {
+  public get name(): string {
+    return `${commands.OUTLOOK_USAGE_GETMAILBOXUSAGESTORAGE}`;
+  }
+
+  public get usageEndpoint(): string {
+    return 'getMailboxUsageStorage';
+  }
+
+  public get description(): string {
+    return 'Get the amount of storage used in your organization.';
```

Comment:
Could you please add a `mailbox` in the sentence? For me, it was not clear what kind of storage is about when I first opened the PR.